### PR TITLE
Add assign action for Product Roadmap

### PR DESCRIPTION
**Pull request by tayloramurphy**
This PR adds the https://github.com/marketplace/actions/assign-to-one-project GitHub action to assign all new PRs and Issues to the Product Roadmap project https://github.com/orgs/meltano/projects/4/views/2

### CODE REVIEW

File: .github/workflows/assign-to-projects.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Auto Assign to Project(s)
+
+on:
+  issues:
+    types: [opened]
+  pull_request_target:
+    types: [opened]
+env:
+  MY_GITHUB_TOKEN: ${{ secrets.MY_GITHUB_TOKEN }}
```
[the remark this answers is not in the document]
Comment (tayloramurphy) [reply .github/workflows/assign-to-projects.yml:9]: @pandemicsyn I actually don't know what secrets we have in place. I'd assumed there's a default secret I could use but do I have to make a personal one and add that? Or we could use MeltyBot I suppose!

[the remark this answers is not in the document]
Comment (edgarrmondragon) [reply .github/workflows/assign-to-projects.yml:9]: @tayloramurphy `secrets.GITHUB_TOKEN` should be available automatically in actions, so you should be able to use that

https://docs.github.com/en/actions/security-guides/automatic-token-authentication

[the remark this answers is not in the document]
Comment (aaronsteers) [reply .github/workflows/assign-to-projects.yml:9]: We have ability to make the github token readonly or read/write. I think we have it as read/write but lmk if you get access denied.

**Review by aaronsteers — APPROVED** [review]

File: .github/workflows/assign-to-projects.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+jobs:
+  assign_one_project:
+    runs-on: ubuntu-latest
+    name: Assign to Product Roadmap Project
+    steps:
+    - name: Assign NEW issues and NEW pull requests to Product Roadmap
+      uses: srggrs/assign-one-project-github-action@1.2.1
+      if: github.event.action == 'opened'
```
[the remark this answers is not in the document]
Comment (tayloramurphy) [reply .github/workflows/assign-to-projects.yml:18]: @aaronsteers I'm fine with either. If it makes a lot of noise I think we would just remove lines 6 & 7.

[the remark this answers is not in the document]
Comment (aaronsteers) [reply .github/workflows/assign-to-projects.yml:18]: Thanks!

**Comment by tayloramurphy** [comment]
@aaronsteers as this is a workflow only change I'm going to merge even with the failing test.

**Pull request merged**